Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.75.0-bullseye AS builder
+RUN apt-get update && apt-get -y upgrade && apt-get install -y make libclang-dev
+COPY . grandine
+RUN cd grandine && ./scripts/build/release.sh
+
+FROM ubuntu:latest
+COPY --from=builder /usr/local/cargo/bin/grandine /usr/local/bin/grandine
```

Comment:
you might wanna set the default entrypoint too.

Reply:
Just wondering is there any other benefits of default entrypoint apart from the fact that user doens't need to write "grandine" when running container?

Reply:
no, thats about it :D I usually assume that container --help will give me help, and wouldn't need to find the binary.